core/src/main/java/com/linecorp/armeria/common/multipart/Multipart.java

```diff
@@ -140,6 +142,22 @@ static Multipart from(HttpRequest request) {
     /**
      * Returns a decoded {@link Multipart} from the the specified {@code boundary} and
      * {@link Publisher} of {@link HttpData}.
+     * For instance, {@link Multipart} could be decoded from the specified {@link HttpResponse}
+     * in the following way:
+     * <pre>{@code
+     * HttpResponse response = ...;
+     * SplitHttpResponse splitResponse = response.split();
+     * ResponseHeaders responseHeaders = splitResponse.headers().join();
+     * StreamMessage<HttpData> responseContents = splitResponse.body();
+     * MediaType contentType = responseHeaders.contentType();
+     * if (contentType != null && contentType.isMultipart()) {
+     *     String boundary = Multiparts.getBoundary(contentType);
+     *     Multipart multipart = Multipart.from(boundary, responseContents)
+     *     ...
+     * } else {
+     *     handleNonMultipartResponse(response);
```

Comment:
nit: Let's do `handleNonMultipartResponse(responseContents)` or `handleNonMultipartResponse(splitResponse)`
because we cannot subscribe to the `response` anymore.

Reply:
Thanks for the tip! I will change this